test/unit/init/AppRunner.test.ts

```diff
@@ -90,6 +118,41 @@ describe('AppRunner', (): void => {
       expect(cliExtractor.handleSafe).toHaveBeenCalledTimes(0);
       expect(settingsResolver.handleSafe).toHaveBeenCalledTimes(0);
       expect(app.start).toHaveBeenCalledTimes(0);
+      expect(app.clusterManager.isSingleThreaded()).toBeFalsy();
+    });
+
+    it('throws an error if there are threading issues detected.', async(): Promise<void> => {
+      errorOnSingleThreadedComponentsMock.mockImplementationOnce((): void => {
+        throw new Error('Threading issue');
+      });
+      const variables = {
+        'urn:solid-server:default:variable:port': 3000,
+        'urn:solid-server:default:variable:loggingLevel': 'info',
+        'urn:solid-server:default:variable:rootFilePath': '/var/cwd/',
+        'urn:solid-server:default:variable:showStackTrace': false,
+        'urn:solid-server:default:variable:podConfigJson': '/var/cwd/pod-config.json',
+        'urn:solid-server:default:variable:seededPodConfigJson': '/var/cwd/seeded-pod-config.json',
+      };
+
+      let caughtError: Error = new Error('should disappear');
```

Comment:
```suggestion
      let caughtError: Error | undefined;
```
Was slightly confused first what this error was doing here